coral-component-colorinput/src/scripts/ColorInputSwatch.js

```diff
@@ -107,10 +109,12 @@ const ColorInputSwatch = Decorator(class extends BaseColorInputAbstractSubview(B
     if (cssColorValue) {
       this._elements.colorButton.style.backgroundColor = cssColorValue;
       this._elements.colorButton.label.textContent = hexColorValue;
+      this._elements.colorLabel.textContent = hexColorValue;
       this.setAttribute('data-value', hexColorValue);
     } else {
       this._elements.colorButton.classList.add('_coral-ColorInput-swatch-novalue');
       this._elements.colorButton.label.textContent = i18n.get('unset');
+      this._elements.colorLabel.textContent = i18n.get('unset');
```

Comment:
Using coral-tooltip we set the textContent of the content element.
```suggestion
      this._elements.colorLabel.content.textContent = i18n.get('unset');
```